src/popup/components/FocusView.tsx

```diff
@@ -189,6 +189,52 @@ export const FocusView = ({ userId }: { userId: string }) => {
 		return <ConnectAProvider />;
 	}
 
+	let content: JSX.Element | null = null;
+	if (focusViewDataQuery.isLoading) {
+		content = (
+			<div className="text-center">
+				<i className="fa-regular fa-spinner-third fa-spin" />
+			</div>
+		);
+	} else if (focusViewDataQuery.error) {
+		const error = focusViewDataQuery.error as FocusViewDataError;
+		content = (
+			<div className="focus-view-error text-secondary">
+				<div className="italic">
+					An unexpected error occurred trying to load Launchpad data.{' '}
+					{error.domain ? (
+						<>
+							Make sure the service at{' '}
+							<ExternalLink className="text-link" href={error.domain}>
+								{error.domain}
+							</ExternalLink>{' '}
+							is reachable. If it is, make sure your access token is still valid.
```

Comment:
IMO this message is a bit long. Normally I'd want to split it into two cases, one is the domain isn't reachable, and another is the access token is bad.

However I see that we can't distinguish that with the errors thrown by the provider-api package, so this is fine as-is.